rules.md

```diff
@@ -391,89 +342,49 @@ Admins must judge that Concepts they wish to Accept meet all of the following cr
 
-> In Trello, Ownership is determined by the Admin who is a Member of the Card and who's username isn't specified in the Secondary Admin field.  
+### 4.2.3 Assistant Admins
 
-### 4.2.3 Secondary Admins
+The Strategist can optionally appoint an assistant Admin to the Bounty, enabling a maximum of two Admins to manage a bounty at one time. Assistant admins can edit the Bounty's details and carry out all administrative actions, but the Bounty is owned by the Strategist.  Assistant Admin rewards are described in [Admin Rewards](#44-admin-rewards).
 
-The Bounty Owner can optionally appoint a Secondary Admin (Secondary) to the Bounty, enabling a maximum of two Admins to manage a bounty at one time.
-
-Secondaries cannot edit the Bounty's details but can create Tasks that they [own](#432-task-ownership) within the Bounty independently from the Bounty Owner's Tasks, and process Claims on those and receive their Admin Reward.
-
-This enables multiple Admins to manage a single Bounty and complete Tasks provided that those Tasks are owned by another Admin on the Bounty.
-
-The Bounty's Owner Admin can transfer the Secondary slot to a different Admin at any time, or remove the Secondary altogether without their agreement, however, the removed Secondary will still retain control of any Tasks they owned on the Bounty.  
-
-Secondaries can leave a Bounty or transfer the secondary position to another Admin who agrees at any time.
-
-> In Trello, Bounty Owners can appoint Secondaries on a Card by entering their name in the Secondary Admin field and assigning them as a Member of the Card. 
-
-### 4.2.4 Forking Bounties
-
-Apart from creating a new Bounty from a Concept, Admins have another option to fork a Bounty, by creating a new Bounty that shares the original Concept but is providing some significant modification, revision or alternative implementation to the Bounty that is being forked.
-
-> In Trello, a fork can be created from a Bounty's Card by just making a new Card and linking the Concept that both Card's share in the Concept Doc custom field of the card.  
+> In Trello, Bounty Owners can appoint an assistant on a Card by entering their name in the assistant Admin field and assigning them as a Member of the Card. 
 
 ## 4.3 Task Management
 
 ### 4.3.1 Creating Tasks
 
-Admins can create [Tasks](#25-tasks) on any Bounty that they're the current Owner or Secondary Admin of.
-
-When creating a Task, the Admin sets the Description, which must provide enough information to allow a Contributor to complete the work, either in the description itself or by linking a Specification.
+Admins can create [Tasks](#25-tasks) on any Bounty that they own or are an Assistant Admin on.  When creating a Task, the Admin sets the Description, which must provide enough information to allow a Contributor to complete the work, either in the description itself or by linking a Specification.
```

Comment:
Strategists own Bounties, not Admins?

Reply:
Fixed in https://github.com/dashincubator/dash-incubator-rules/pull/11/commits/c4c94a5e12f23b7efe0e79e6b4c694488486ce9b